OracleDatabase/samples/prebuiltdb/README.md

```diff
@@ -0,0 +1,98 @@
+# Example of creating an image with pre-built DB
+=================================================
+Warning! The description below requires changes in the dockerfile related to the version and edition in question (docker-images/OracleDatabase/dockerfiles/$VERSION/Dockerfile.$EDITION). It's recommended that you revert the changes after you have completed creating image without VOLUME (see next section).
+
+## 1. Create an image without a VOLUME
+
+When creating an image with a pre-build DB, we first need an image without a VOLUME. To create such an image you'll need to comment out the VOLUME command in the dockerfile in use (docker-images/OracleDatabase/dockerfiles/$VERSION/Dockerfile.$EDITION).
+
+Comment out the following line:
+```
+#VOLUME ["$ORACLE_BASE/oradata"]
+```
+In the example below I have edited the Dockerfile for version 12.1.0.2 and Enterprise Edition (ee) (docker-images/OracleDatabase/dockerfiles/12.1.0.2/Dockerfile.ee).
+
+If you want to keep your existing image, you need to rename it:
+```
+docker tag oracle/database:12.1.0.2-ee oracle/database-org:12.1.0.2-ee
+```
+
+Then create an image without a VOLUME:
+```
+cd docker-images/OracleDatabase/dockerfiles
+sh buildDockerimage.sh -v 12.1.0.2 -e -i 
+```
+Note! When the build completes remember to uncomment the VOLUME command in the current dockerfile.
+
+## 2. Startup a container and create the database
+
+We first need to start up a container to get the database created.
+Note!  First make sure you have built **oracle/database:12.1.0.2-ee**. 
+Now start the container as follow:
+```
+docker run --name oracle-build -p 1521:1521 -p 5500:5500 oracle/database:12.1.0.2-ee
+```
+
+## 3. Reset passwords (optional)
+
+It's recommended to reset passwords before creating the image. This way you don't have to do it everytime you create a new container.
+
+Now you should connect to the container and reset passwords:
+```
+docker exec -it oracle-build setPassword.sh <newPassword>
```

Comment:
This has to have a leading `./` as the current directory isn't in the PATH. It also doesn't need the `-it` as this is not interactive any longer: `docker exec oracle-build ./setPassword.sh LetsDocker`